### PR TITLE
Improve driver cleanup with shutdown hook

### PR DESCRIPTION
**Pull request by mrk-han**
SauceLabs warned us that we may not be properly cleaning up our sessions. This PR will help us quit the driver even when receiving a SIGINT or another unexpected crash. 

We're using `java.lang.Runtime.addShutdownHook()` to ensure Appium receives a quit message even if the process is interrupted. 

Cleaning up our sessions will help idle concurrency which will remove some of the execution bottleneck. 

One edge case that is unaccounted for is if we request a session and the test runner dies before it's created. In this case we will have to wait for the timeout to occur, which was reduced drastically from 10 minutes to 90 seconds.

### CODE REVIEW

File: src/main/java/com/joss/conductor/mobile/Locomotive.java [hunk 1]
```diff
@@ -827,14 +825,17 @@ public Locomotive longPressSwipeLong(SwipeElementDirection direction, WebElement
         return longPressSwipe(direction, webElement, SWIPE_DISTANCE_LONG, 0);
     }
 
+    @Deprecated
     public Locomotive longPressSwipeLong(SwipeElementDirection direction, String id, int swipeDurationInMillis) {
```
Comment (evant) [inline src/main/java/com/joss/conductor/mobile/Locomotive.java]: All deprecated methods really should have some javadoc on why they are deprecated and what you should do about it.

Comment (mrk-han) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: I think @aaron-goff deprecated these and I dunno the reasoning :\ I just added these because the other method had the annotation

Comment (aaron-goff) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: 🤔 I'm not sure that I deprecated these? I think these ones are the ones that shouldn't be deprecated?

Comment (mrk-han) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: Ohh I see -- you deprecated the one that takes in a webelement -- i'll add a comment

Comment (mrk-han) [reply src/main/java/com/joss/conductor/mobile/Locomotive.java]: Fixed in ef29718
